src/app/[locale]/viewer/_components/submodel-elements/generic-elements/entity-components/EntityTreeItem.tsx

```diff
@@ -44,7 +49,7 @@ const CustomContent = React.forwardRef(function CustomContent(props: CustomTreeI
             // if not, just navigate to the specified URL which might lead anywhere.
 
             const aasIds = await performDiscoveryAasSearch(assetId);
```

Comment:
imo this should be:
`const response = await await performDiscoveryAasSearch(assetId);
if (response.isSuccess && response.result.length == 0)`

Reply:
You are right. Changed